back/lib/spherical_point_impl_active_job_serializer.rb

```diff
@@ -0,0 +1,20 @@
+# https://blog.saeloun.com/2019/09/11/rails-6-custom-serializers-for-activejob-arguments/
+# https://api.rubyonrails.org/classes/ActiveJob/Serializers/ObjectSerializer.html
+class SphericalPointImplActiveJobSerializer < ActiveJob::Serializers::ObjectSerializer
+  def serialize(point)
+    super(
+      'x' => point.x,
+      'y' => point.y
+    )
```

Comment:
Maybe/probably not relevant in our case, but `SphericalPointImpl` also have z and m coordinates.

Reply:
We seem to use only two points.

Also, `RGeo::Geographic.spherical_factory.point` accepts only two arguments. Maybe there's another way of deserializing it, but I'm not aware of it.

Reply:
I think there are different factories for `SphericalPointImpl`.

Reply:
Couldn't find it so far. But let me look again.
```
[23] pry(main)> RGeo::Geographic.spherical_factory.point(1, 2)
=> #<RGeo::Geographic::SphericalPointImpl:0x37514 "POINT (1.0 2.0)">
[24] pry(main)> RGeo::Geographic.methods.grep /factory/
=> [:spherical_factory, :simple_mercator_factory, :projected_factory]
```

Reply:
Apparently, we can do something like that:
```ruby
RGeo::Geographic.spherical_factory({
  has_z_coordinate: true,
  has_m_coordinate: true,
}).point(...)
```
but it's not that important.

Reply:
It turned out that our columns don't use `m` and `z`.
```ruby
[63] pry(main)> Event.columns.find { _1.name == 'location_point' }.has_m?
=> false
[64] pry(main)> Event.columns.find { _1.name == 'location_point' }.has_z?
=> false
```
I wasn't sure what factory to use and then decided to support both cases https://github.com/CitizenLabDotCo/citizenlab/pull/8412/commits/b0cbad5550c2a30f53b6867495c376ef0648cfa6